modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -775,11 +893,13 @@ PERF_TEST_P_(BitwisePerfTest, TestPerformance)
 
 PERF_TEST_P_(BitwiseNotPerfTest, TestPerformance)
 {
-    cv::Size sz_in = get<0>(GetParam());
-    MatType type = get<1>(GetParam());
-    cv::GCompileArgs compile_args = get<2>(GetParam());
+    compare_f cmpF;
+    cv::Size sz;
+    MatType type;
```

Comment:
Please add initialization with default value for `type`. See example in the `MulPerfTest` case (above).

Reply:
Applied, thanks